src/components/remote-services/endpoint-form/EndpointForm.jsx

```diff
@@ -263,6 +265,10 @@ const EndpointForm = ({ initialValues, handleSubmit, serviceURL }) => {
                 header='Advanced'
                 key='1'
               >
+                <FormItemLabel name="Timeout" description="default: 60" />
+                <Form.Item name="timeout">
+                  <InputNumber style={{ width: "100%" }} placeholder="Timeout in seconds" />
```

Comment:
Seems like you haven't imported `InputNumber` from `antd`. The code is not building.